.circleci/config.yml

```diff
@@ -149,8 +149,9 @@ jobs:
     executor: win64-executor
     steps:
       - checkout
+      - run: dpkg --add-architecture i386
       - run: apt update
-      - run: DEBIAN_FRONTEND=noninteractive apt install -y ninja-build python3 python3-pip python3-venv git g++-mingw-w64-x86-64 zip wine wine64
+      - run: DEBIAN_FRONTEND=noninteractive apt install -y ninja-build python3 python3-pip python3-venv git g++-mingw-w64-x86-64 zip wine wine64 wine32:i386
```

Comment:
That is odd, why is it complaining about wine32 now? May be we should be using `wine64` in win64-cross-file.txt?

Reply:
It might have been to run `dir`.